eng/build.yml

```diff
@@ -34,3 +24,11 @@ steps:
     command: test
     arguments: -c $(BuildConfiguration) --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include="[coverlet.collector]*%2c[coverlet.core]*%2c[coverlet.msbuild.tasks]*" /p:Exclude="[coverlet.core.tests.samples.netstandard]*%2c[coverlet.tests.xunit.extensions]*"
     testRunTitle: $(Agent.JobName)
+
+- template: publish-coverlet-result-files.yml
+
+#- template: publish-coverage-results.yml
```

Comment:
Why is this commented? I also like this idea of having an html report that can be looked into. Unfortunately I don't have much control over the pipeline and thus I can't tell anything about file restrictions...

Reply:
The template uses `reportgenerator@5` task which is currently not available on Azure DevOps **tonerdo** organization
(see #1516).